boards/stm32f4discovery/dist/debug.sh

```diff
@@ -1,4 +1,19 @@
 #!/bin/sh
 
-echo "Debugging $1"
-arm-none-eabi-gdb -tui -command=$1 $2
+openocd -f "${RIOTBOARD}/${BOARD}/dist/openocd.cfg" \
```

Comment:
For this and all other `*.sh`s, in current openocd versions there exists `board/stm32f4discovery.cfg`.